app/retail/utils.py

```diff
@@ -151,6 +151,9 @@ def get_completion_rate(keyword):
     not_completed_bounties = eligible_bounties.filter(idx_status__in=['expired', 'cancelled']).count()
     total_bounties = completed_bounties + not_completed_bounties
 
+    if total_bounties == 0:
```

Comment:
Any chance we can go the more idiomatic route ([EAFP](https://docs.python.org/3.7/glossary.html#term-eafp)) here and simply catch the exception of `ZeroDivisionError` with a try/except?  In any case, if you went with the lbyl route, you could check `if not`.

Reply:
Sure, let me do it the EAFP way.